server/views/pages/profile.njk

```diff
@@ -71,14 +74,18 @@
         {% endif %}
       </div>
     </div>
-
+      
```

Comment:
Linter would normally have picked this up 🤔

Reply:
Yeh, true 🤔